scripts/support/README.rst

```diff
@@ -45,6 +45,9 @@ the support account, encrypt the password and print the encrypted password to co
 Installation
 ============
 
+Some scripts of this folder use *osclients.py*. That file can be copied from
+*utils/osclients.py*.
```

Comment:
Ok, I'll change  osclients -> import utils/osclients and change this in the documentation:

Some scripts of this folder imports *utils/osclients.py*. Therefore the root project
should be included in PYTHONPATH.